### DISCUSSION TITLE
Conteúdo gratuito!

### DISCUSSION THREAD

**Discussion by YasmimVieira** [frontendbr/forum#2010] [Mostre seu trabalho]
Eai galera, beleza? Eu sou desenvolvedora a algum tempo e decidi compartilhar alguns conhecimentos, então estou criando artigos no medium sobre conteúdos web, principalmente frontend! Meu foco é em algumas questões básicas, que por mais que simples, muitas pessoas pulam essas etapas que são importantíssima.

### **O meu medium é esse aqui:** https://yasmimv.medium.com/

Espero que vocês possam acompanhar, estou tentando pegar um público e trabalhar com ele, espero que gostem dos conteúdos e todo feedback é bem vindo!

![971](https://user-images.githubusercontent.com/47395019/106811410-0d80ec80-664d-11eb-9200-b926566667ad.jpg)

**Comment by fabiochiquezi** [comment]
Show, vou começar a seguir!

**Reply by YasmimVieira** [reply]
Muito obrigada!

**Comment by jardelima** [comment]
Conteúdo muito massa!

**Reply by YasmimVieira** [reply]
Muito obrigada mesmo!

**Comment by TheyCallMeErick** [comment]
Seguindo ihihihi

**Comment by vagner531** [comment]
Show o conteúdo!

**Comment by joaroque** [comment]
Incrível amigo
